cityofaustin/atd-moped
# disable write permissions primary keys

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/12246

## Testing
**URL to test:** 

local

**Steps to test:**

i made these changes manually by examining the `update` and `insert` permissions for all tables in `tables.yaml` and cross-referencing those with the table's primary key. if the primary key showed up in any of these cases for any role i slashed it.

- afaik the best way to test this is to read through the deletions in `tables.yaml` and confirm that they are all related to `update` or `insert` permissions. alternatively, you can go through the hasura console by clicking through the permissions on each table but i think that would be much more time-consuming

---
#### Ship list
